electricitymap/contrib/lib/tests/test_event_lists.py

```diff
@@ -50,26 +50,27 @@ def test_append_to_list_logs_error():
 
 
 def test_merge_exchanges():
+    dt = datetime(2023, 1, 1, tzinfo=timezone.utc)
     exchange_list_1 = ExchangeList(logging.Logger("test"))
     exchange_list_1.append(
         zoneKey=ZoneKey("AT->DE"),
-        datetime=datetime(2023, 1, 1, tzinfo=timezone.utc),
+        datetime=dt,
         netFlow=1,
         source="trust.me",
     )
     exchange_list_2 = ExchangeList(logging.Logger("test"))
     exchange_list_2.append(
         zoneKey=ZoneKey("AT->DE"),
-        datetime=datetime(2023, 1, 1, tzinfo=timezone.utc),
+        datetime=dt,
         netFlow=2,
         source="trust.me",
     )
     exchanges = ExchangeList.merge_exchanges(
         [exchange_list_1, exchange_list_2], logging.Logger("test")
     )
     assert len(exchanges) == 1
-    assert exchanges.events[0].datetime == datetime(2023, 1, 1, tzinfo=timezone.utc)
-    assert exchanges.events[0].netFlow == 3
+    assert exchanges[dt].datetime == datetime(2023, 1, 1, tzinfo=timezone.utc)
+    assert exchanges[dt].netFlow == 3
```

Comment:
This was the easiest way to test that the types where properly passed down. the datetime and netFlow types where properly recognized! 🙌🏻